src/main/java/com/google/javascript/gents/ClassConversionPass.java

```diff
@@ -109,27 +109,29 @@ public void visit(NodeTraversal t, Node n, Node parent) {
     public void visit(NodeTraversal t, Node n, Node parent) {
       if (n.isExprResult()) {
         ClassMemberDeclaration declaration = ClassMemberDeclaration.newDeclarationOnThis(n);
-        // Ignore field declarations without a type annotation
-        if (declaration != null &&
-            declaration.jsDoc != null &&
-            declaration.jsDoc.getType() != null) {
-          Node fnNode = NodeUtil.getEnclosingFunction(n);
-          String fnName = fnNode.getParent().getString();
-          JSTypeExpression type = declaration.jsDoc.getType();
-          // All declarations of the form this.name = name;
-          if ("constructor".equals(fnName) && declaration.rhsEqualToField()) {
-            Node params = fnNode.getSecondChild();
-            JSDocInfo constructorJsDoc = NodeUtil.getBestJSDocInfo(fnNode);
-
-            for (Node param : params.children()) {
-              JSTypeExpression paramType = constructorJsDoc.getParameterType(param.getString());
-              // Names and types must be equal
-              if (declaration.memberName.equals(param.getString()) && type.equals(paramType)) {
-                // Add visibility directly to param if possible
-                moveAccessModifier(declaration, param);
-                n.detachFromParent();
-                compiler.reportCodeChange();
-                return;
+
+        if (declaration != null) {
+          // TODO(gmoothart): in many cases we should be able to infer the type from the rhs
```

Comment:
I played around with this a little but couldn't get anything working. I couldn't figure out how to convert (for example) a rhs that is a NUMBER to a JSTypeExpression needed by the code below.

There are also cases like
```
this.a = getFoo();
```
where we know the return type of getFoo. And in this case too I couldn't figure out how to get the AST to give me what I wanted.